plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -1764,7 +1764,7 @@ public String toString() {
                 graphicBuilder.append(" listen=''");
             }
             if (_passwd != null) {
-                graphicBuilder.append(" passwd='" + _passwd + "'");
+                graphicBuilder.append(" passwd='" + StringUtils.truncate(_passwd, 8) + "'");
```

Comment:
Why is that @weizhouapache ? Does this fix/address any libvirt limitation or user reported issue?

Reply:
@rohityadavcloud 
this aims to fix an issue with ubuntu 22.04 (to be released soon) and libvirt 8.
the issue is raised in mailing list: https://lists.apache.org/thread/zk4fyggjqk1zphk4dfg1m596yrd15d9b

from many years ago, libvirt accepts vnc password with more than 8 chars. but only the first 8 chars in vnc password are used in vnc console verification (the other chars are dropped).
In recent libvirt, only vnc password with 8 chars is permitted.

Reply:
LGTM